spec/requests/government_ruleset_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe GovernmentRulesetController, type: :controller do
```

Comment:
Best practice I believe is to use request specs over controller specs. However, if the behaviour really has to be tested via a controller spec then that spec should be `spec/controllers` rather than `spec/requests`.

Reply:
this is a silly error on my part. It should be a request spec not a controller spec.